### PR TITLE
Add hackathon prizes view and projects

### PR DESCRIPTION
**Pull request by zoek1**
##### Description
- [X] Implement the headers and sidebar of the new hackathons design
- [X] Implement design for prizes list
- [X] Implement projects views

##### Refers/Fixes
No issue

### Hackathon prizes
Image Pasted at 2020-3-30 01-00

### Projects view
Image Pasted at 2020-3-31 01-52

### CODE REVIEW

File: app/dashboard/templates/dashboard/sidebar_search_hackathon.html [hunk 1]
```diff
@@ -33,17 +53,30 @@
...
-
+        <div class="hackathon-logo mb-1">
+          {% if hackathon.logo_svg %}
+            <img class="d-none d-md-block mx-auto mb-5 mw-100" src="{{ hackathon.logo_svg.url }}" alt="Hackathon logo"/>
+          {% elif hackathon.logo %}
+            <img class="d-none d-md-block mx-auto mb-1 mw-100" src="{{ hackathon.logo.url }}" alt="Hackathon logo"/>
+          {% endif %}
+        <small>{{ hackathon.start_date|date:"M j, Y" }} - {{ hackathon.end_date|date:"M j, Y" }}</small>
+        </div>
+        <a href="/{{hackathon.town_square_link}}" class="btn btn-gc-blue register col-xs-12 mt-4 mb-2">
```
Comment (octavioamu) [inline app/dashboard/templates/dashboard/sidebar_search_hackathon.html:64]: what this "register" button do?

Comment (zoek1) [reply app/dashboard/templates/dashboard/sidebar_search_hackathon.html]: Redirects to the hackathon onboarding

Comment (octavioamu) [reply app/dashboard/templates/dashboard/sidebar_search_hackathon.html]: cool

**Comment by octavioamu** [comment]
How projects explorer queries and paginations are being handled here? since use to work with url params

**Comment by octavioamu** [comment]
code merged already in  #6359

**Pull request merged**